tests/test_secret.py

```diff
@@ -47,19 +48,37 @@
 ]
 
 
-def test_secret_box_creation():
-    SecretBox(
-        b"ec2bee2d5be613ca82e377c96a0bf2220d823ce980cdff6279473edc52862798",
-        encoder=HexEncoder,
-    )
+def hex_key(m):
+    return st.binary(min_size=m.KEY_SIZE, max_size=m.KEY_SIZE).map(binascii.hexlify)
 
 
-def test_secret_box_bytes():
-    s = SecretBox(
-        b"ec2bee2d5be613ca82e377c96a0bf2220d823ce980cdff6279473edc52862798",
-        encoder=HexEncoder,
-    )
-    assert bytes(s) == s._key
+def box(m):
```

Comment:
Let's name this something else just to minimize confusion since there are also args below that are named box